packages/ui-badge/src/Badge/__tests__/Badge.test.js

```diff
@@ -68,19 +66,6 @@ describe('<Badge />', () => {
     expect(await badge.find(':contains(99 +)')).to.exist()
   })
 
-  it('should change postion based on the placement prop', async () => {
-    await mount(
-      <Badge count={3} placement="bottom start">
-        <button type="button">Inbox</button>
-      </Badge>
-    )
-
-    const badge = await BadgeLocator.find()
-
-    expect(await badge.find(`.${styles['positioned--bottom']}`)).to.exist()
-    expect(await badge.find(`.${styles['positioned--start']}`)).to.exist()
-  })
-
```

Comment:
IMO we should just comment out with a TODO comment tests that are hard to port

Reply:
I removed it, because these are the tests that will be better covered in VRT.

Reply:
since I had to modify the commit anyway, I put it back with a comment :)